CREDITS.TXT

```diff
@@ -73,3 +73,4 @@ Huitao Chen (chenhuitao) & KaiJern Lau (xwings): Cmake support
 Huitao Chen (chenhuitao) & KaiJern Lau (xwings): Python3 support for building
 Kevin Foo (chfl4gs): Travis-CI migration
 Ziqiao Kong (lazymio): uc_context_free() API and various bug fix & improvement.
+Sven Almgren (blindmatrix)
```

Comment:
Need a more detailed description here? Not sure.

Cc @aquynh

Reply:
I added ": bug fix"